andela/firestar-backend
# #168371026 Fix staging deployment

#### What does this PR do?
  Adds Clean script to package.json file
  Adds build script to package.json file
  Adds heroku-postbuild script to package.json file

#### Description of Task to be completed?
  Make changes to the app to ensure it runs in the staging environment

#### How should this be manually tested?
  Open postman, make a get request to `https://firestar-backend-staging-pr-38.herokuapp.com/api/v1/users/email/verify` with an email in the request body. An appropriate message will be received
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168371026](https://www.pivotaltracker.com/story/show/168371026)

#### Screenshots (if appropriate)
![Screenshot (56)](https://user-images.githubusercontent.com/46751363/64544509-09994f80-d31f-11e9-86a4-188cdfecc750.png)
![Screenshot (57)](https://user-images.githubusercontent.com/46751363/64545551-db1c7400-d320-11e9-99e0-1cdb7dd71f59.png)
![Screenshot (58)](https://user-images.githubusercontent.com/46751363/64545559-df489180-d320-11e9-868f-808549d905d4.png)
![Screenshot (59)](https://user-images.githubusercontent.com/46751363/64545567-e374af00-d320-11e9-97e5-65dd6be07590.png)


#### Questions:
N/A
